tests/helpers.py

```diff
@@ -982,3 +995,99 @@ def delete_deploymentconfig(pod_obj):
     """
     dc_ocp_obj = ocp.OCP(kind=constants.DEPLOYMENTCONFIG)
     dc_ocp_obj.delete(resource_name=pod_obj.get_labels().get('name'))
+
+
+def delete_objs_parallel(obj_list):
+    """
+    Deletes objs specified in list
+    Args:
+        obj_list(list): List can be obj of pod, pvc, etc
+
+    Returns:
+        bool: True if obj deleted else False
+
+    """
+    with ThreadPoolExecutor(32) as p:
+        for obj in obj_list:
+            p.submit(obj.delete)
+    return True
+
+
+def memory_leak_analysis(median_dict):
+    """
+    Memory leak analysis after the execution of test case
+    Memory leak is analyzed based on "RES" value, i.e. list[7] in code
+    Args:
+         median_dict (dict): dict of worker nodes and respective median value
+    """
+    # dict to store memory leak difference for each worker
+    diff = {}
+    for worker in get_worker_nodes():
+        memory_leak_data = []
+        if os.path.exists(f"/tmp/{worker}-top-output.txt"):
+            with open(f"/tmp/{worker}-top-output.txt", "r") as f:
+                data = f.readline()
+                list = data.split(" ")
+                list = [i for i in list if i]
+                memory_leak_data.append(list[7])
+        else:
+            logging.info(f"worker {worker} memory leak file not found")
+            raise UnexpectedBehaviour
+        number_of_lines = len(memory_leak_data) - 1
+        # Get the start value form median_dict arg for respective worker
+        start_value = median_dict[f"{worker}"]
+        end_value = memory_leak_data[number_of_lines]
+        logging.info(f"Median value {start_value}")
+        logging.info(f"End value {end_value}")
+        # Convert the values to kb for calculations
+        if start_value.__contains__('g'):
+            start_value = float(1024 * 1024 * float(start_value[:-1]))
```

Comment:
Please create constant for 1024 * 1024 called MB
Also, you can do:
MB = 1024 ** 2